slyd/slyd/splash/css_utils.py

```diff
@@ -1,12 +1,37 @@
 import re
 import urllib
+import six
+import htmlentitydefs
 from six.moves.urllib_parse import urlparse, urljoin
 
-
 CSS_IMPORT = re.compile(r'''@import\s*["']([^"']+)["']''')
 CSS_URL = re.compile(r'''\burl\(("[^"]+"|'[^']+'|[^"')][^)]+)\)''')
 BAD_CSS = re.compile(r'''(-moz-binding|expression\s*\(|javascript\s*:)''', re.I)
 
+# https://html.spec.whatwg.org/multipage/syntax.html#character-references
+# http://stackoverflow.com/questions/18689230/why-do-html-entity-names-with-dec-255-not-require-semicolon
+_ENTITY_RE = re.compile("&#?\w+;", re.I)
+def _replace_entity(match):
+    entity = match.group(0)
+    if entity[:2] == "&#":
+        # character reference
+        if entity[:3] == "&#x":
+        if entity[:3] == "&#x":
```

Comment:
FYI there has been a merging error on my side. This line should not have been doubled...